features/step_definitions/account_creation.rb

```diff
@@ -10,54 +10,61 @@
   expect(page).to have_content "Log in"
 end
 
-When(/^I click "(.*?)"$/) do |arg1|
-  pending # express the regexp above with the code you wish you had
+When(/^I click the "(.*?)" link$/) do |arg1|
+  click_link arg1 
 end
 
-When(/^I click the button "(.*?)"$/) do |arg1|
+When(/^I click the "(.*?)" button$/) do |arg1|
   click_button arg1
 end
 
 When(/^I enter my email address$/) do
-  pending # express the regexp above with the code you wish you had
+ fill_in :user_email, with: 'test@example.com' 
```

Comment:
This isn't something everyone's doing yet, but I think works well. Feel free to not do this if you want:

Instead of hardcoding the email address here, change your "I have an account" step to set up `@user` or even `@my`. Then here, you can do `@user.email` or `@my.email`